source/includes/reviews-for-all-apps.md

```diff
@@ -178,9 +178,10 @@ Field                    | Description
 **app_version**          | The app version this review is for, if present. ![itunes_only](../images/itunes_logo.jpg)
 **date**                 | Review date as a string in ISO format: `YYYY-MM-DD`.
 **date_str**             | The original review date format, a string. ![android_only](../images/android_logo.jpg)
+**id**                   | The globally unique reviews ID. For iOS apps, we use IDs provided by App Store and for Android apps we generate them on our end by calculating a `md5_hash` on the following fields: `app_id`, `review_text`, `user_name`, `rating`, `title`, `language`.
```

Comment:
Please refer to iOS/iTunes apps as "itunes apps" and for Android apps as "android apps". This is consistent with our `store=itunes|android` values.

Reply:
Make sentences shorter & a few typos: For `itunes` apps, we use ID provided by App Store. For `android` apps we generate them by calculating MD5 hash from: `app_id`, `review_text`, `user_name`, `rating`, `title`, `language`.

Reply:
https://github.com/AppMonsta/slate/commit/ddd2a96deb6f898a62b2009a46485e063bf35701